### ISSUE TITLE
[Feature] Re-tune monitoring: round-aware progress display with boundary expansion panel

### ISSUE THREAD

**Issue by nbx-liz** [nbx-liz/LizyML-Widget#101]
## Summary

LizyML H-0068 introduces re-tune (Study Resume + Boundary Expansion) functionality. The Widget needs to consume the new `TuneProgressInfo` fields to provide round-aware real-time monitoring.

### TuneProgressInfo (real-time callback)

| Field | Type | Description |
|---|---|---|
| `round` | `int` | Current round number (1-indexed) |
| `cumulative_trials` | `int` | Total trials across all rounds |
| `expanded_dims` | `tuple[str, ...]` | Dims expanded in this round |

### TuningResult (final result)

| Field | Type | Description |
|---|---|---|
| `rounds` | `tuple[RoundSummary, ...]` | Per-round summary with scores and expanded dims |
| `boundary_report` | `BoundaryReport \| None` | Boundary detection details |

### 1. Round-aware progress display

- Show current round number (e.g., "Round 2/2")
- Dual progress: round progress + cumulative progress
- Improvement indicator: score delta from previous round's best

### 2. Boundary expansion panel

When `expanded_dims` is non-empty, show a panel:
```
┌─ Boundary Expansion ──────────────────────────────┐
│  learning_rate  ▼ lower  [0.0001→0.00001, 0.1]   │
│  num_leaves     ▲ upper  [16, 256→512]            │
│  3 dims unchanged                                  │
└────────────────────────────────────────────────────┘
```

### 3. Score history chart with round separators

- Vertical dashed line at round boundaries
- Per-round annotation with expanded dim names
- Continuous best-score line across rounds

### 4. Convergence signal

When `expanded_dims` is empty in a resume round:
- Display "No expansion needed — search space is sufficient"
- Suggest proceeding to `fit()`

## Mockup

```
┌─────────────────────────────────────────────────────────┐
│  🔍 Tuning Progress                          Round 2/2  │
│─────────────────────────────────────────────────────────│
│  Trial: 67/80 (Round 2: 17/30)     Elapsed: 3m 42s     │
│  ████████████████████░░░░░░░░░  83%                     │
│  Best Score: 0.2790 (↑ 0.0044 from Round 1)            │
│                                                         │
│  ┌─ Boundary Expansion ────────────────────────────┐    │
│  │  learning_rate  ▼ lower  [0.0001→0.00001, 0.1] │    │
│  │  num_leaves     ▲ upper  [16, 256→512]          │    │
│  │  3 dims unchanged                                │    │
│  └──────────────────────────────────────────────────┘    │
│                                                         │
│  Score History:          Round 1 │ Round 2              │
│  0.32 ┤●                        │                      │
│  0.30 ┤  ●●●                    │                      │
│  0.28 ┤      ●●●●●●─best─ ─ ─ ─│─●●●●●●─new best─    │
│       └─────────────────────────┴────────────── trial   │
└─────────────────────────────────────────────────────────┘
```

## Dependencies

- LizyML >= 0.9.0 (H-0068 merged)

## Related

- LizyML H-0068: Re-tune (Study Resume + Boundary Expansion)
- LizyML BLUEPRINT.md §11.5

**Comment by nbx-liz** [comment]
Released in **lizyml-widget v0.8.0** 🎉

PyPI: https://pypi.org/project/lizyml-widget/0.8.0/
GitHub Release: https://github.com/nbx-liz/LizyML-Widget/releases/tag/v0.8.0

## What shipped

The full re-tune feature requested in this issue is now available, plus the supporting infrastructure:

- **P-027 — Re-tune monitoring**: round-aware ProgressView, Boundary Expansion panel, Convergence Signal banner, lizyml version guard (`lizyml>=0.9.0,<0.10`).
- **P-028 — Re-tune launcher**: `w.retune(n_trials=..., expand_boundary=..., boundary_threshold=...)` Python API and a matching `Re-tune (resume)` button inside the Best Params accordion. The widget reuses the existing Optuna study via a dedicated `_tune_model` slot so an intervening `fit()` does not clobber it.
- **P-029 — Tuning History consolidation**: the Results tab now renders lizyml's `tuning_plot` figure in a dedicated accordion, replacing the widget-local Plotly duplicate (−2.4 KB on the production bundle).
- **Bug fixes** for the ConvergenceSignal display (literal `\\u2713` and off-by-one round number).

All shipped via PR #102 (feature) + PR #104 (release), with 1036 tests (861 pytest + 175 vitest) and clean ruff / mypy / eslint / tsc gates.

## How to use

```bash
pip install --upgrade 'lizyml-widget[lizyml]'
```

```python
from lizyml_widget import LizyWidget

w = LizyWidget()
w.load(df, target='y')
w.tune()                                      # initial tune
w.retune(n_trials=30, expand_boundary=True)   # resume + widen boundaries
```

The Results tab now shows round-aware progress, the boundary expansion panel, lizyml's tuning history plot, and the convergence signal once the optimizer believes the search space is sufficient.

See the [README Re-tune section](https://github.com/nbx-liz/LizyML-Widget/blob/main/README.md#re-tune-study-resume--boundary-expansion) and [docs/VERSION_COMPAT.md](https://github.com/nbx-liz/LizyML-Widget/blob/main/docs/VERSION_COMPAT.md) for the full compatibility matrix and upgrade guide.
